src/main/java/net/projectzombie/region_rotation/modules/RegionState.java

```diff
@@ -36,16 +39,16 @@
     private String rotateBroadcastMessage = null;
 
     public RegionState(final String regionName,
-                       final String worldName)
+                       final UUID worldUID)
     {
-        super(worldName);
+        super(worldUID);
         this.regionName = regionName;
         this.regionType = this.getProtectedRegion().getType();
         this.isValid = super.isValid() && this.regionName != null && this.getProtectedRegion() != null;
     }
 
     /** {@inheritDoc} */
-    @Override public boolean isValid() { return this.isValid; }
+    public boolean isValid() { return this.isValid; }
```

Comment:
Lets add `this.getProtectedRegion() != null` to this function call since the region could get deleted after initialization.